crates/astria-conductor/src/types.rs

```diff
@@ -22,7 +22,7 @@ impl SequencerBlockSubset {
     pub(crate) fn from_sequencer_block_data(
         data: SequencerBlockData,
         namespace: Namespace,
-    ) -> Self {
+    ) -> Option<Self> {
```

Comment:
The change in this signature followes from the removal of the `Default` impl for `RollupData`

Reply:
Note that `RollupData` is again using a `Vec<u8>` as a chain ID. But I am not going to re-add the `Default` impl because it does not make sense to have it.